internal/nginx/manager.go

```diff
@@ -37,6 +37,10 @@ const (
 
 	// appProtectLogConfigFileName is the location of the NGINX App Protect logging configuration file
 	appProtectLogConfigFileName = "/etc/app_protect/bd/logger.cfg"
+
+	appProtectDosAgentInstallCmd    = "/usr/bin/adminstall"
```

Comment:
What does the install command do? Just curious

Reply:
For IPC between admd and nginx